### ISSUE TITLE
Incompatible with Kodi 19, won't display in repository

### ISSUE THREAD

**Issue by eproffitt** [I-A-C/I-A-C.github.io#14]
**Expected Results**
When the user updates to the latest official release of the platform, the addon will function. 

**Actual Results**
When the user updates to the latest official release of the platform (currently 19.00 “Matrix"), the addon has been disabled by the platform for incompatibility. Users with automatic updates set in their app store will probably be the most impacted.

**Steps to Reproduce**
1. _Given that Exodus Redux is installed on Kodi_
2. Update Kodi to release 19.00 "Matrix"
2. Open Kodi
3. Observe Exodus Redux has been disabled

**Comment by eproffitt** [comment]
**Steps to Reproduce**
1. _Given that Exodus Redux is installed on Kodi_
2. Update Kodi to release 19.00 "Matrix"
3. Open Kodi
4. Navigate to Add-ons>Add-on Browser>Install from Repository>{Exodus Redux Repo}
5. Observe the "Video Add-ons" list item is not displayed

**Comment by eproffitt** [comment]
**Steps to Reproduce**
1. _Given that Exodus Redux is installed on Kodi_
2. Update Kodi to release 19.00 "Matrix"
3. Open Kodi
4. Navigate to Add-ons>My Add-ons>Video Add-ons
5. Observe Exodus Redux is marked as disabled
6. Select Exodus Redux
7. Select "Enable"
8. Observe error message from multiple origins stating "This add-on is not compatible with this version of Kodi"

**Comment by TDunnavant** [comment]
This is expected behavior, given that Exodus Redux is based on Python 2.x and Kodi 19 is based on the incompatible Python 3.x.  Exodus Redux no longer is a supported add-on, so it is doubtful this issue will be resolved.

**Comment by tecjlar** [comment]
So is downgrading to Kodi 18.X an option to get Exodus working again?

Thanks

**Comment by eproffitt** [comment]
> So is downgrading to Kodi 18.X an option to get Exodus working again?
> 
> Thanks

Yes, that does work, but you won't be able to update Kodi ever again, considering that Exodus Redux has no support and will probably never be updated to be compatible.

**Comment by tecjlar** [comment]
Is there a video addon that works as well with Kodi 19?

Thanks
